api/commands.go

```diff
@@ -156,6 +156,7 @@ type NewContainer struct {
 	FsType  string  `json:"fstype,omitempty"`
 	Fsmap   []Fsmap `json:"fsmap"`
 	Process Process `json:"process"`
+	Addr    string  `json:"addr,omitempty"`
```

Comment:
Ok here I think we should go with something more comprehensive like `expectedName`, meaning this is the identifier/name that we expect to find inside the VM to find the appropriate device.

Reply:
I have changed this to SCSI address. SCSI address is a string in the format SCSI-Id:LUN. This scsi-id and lun are selected and passed from the host side. I have eliminated the expected name completely with this, to avoid any errors in computing the expected name.